mls_validation_service/src/main.rs

```diff
@@ -24,6 +26,12 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .init();
 
     let args = Args::parse();
+
+    if args.version {
+        println!("Version: {0}", get_version());
```

Comment:
Should be an info! log

Reply:
with `info!` I get this.
```
 $ cargo run --bin mls-validation-service -- -v
   Compiling mls_validation_service v0.1.0 (/Users/martinkysel/work/libxmtp/mls_validation_service)
    Finished `dev` profile [unoptimized] target(s) in 1.69s
     Running `target/debug/mls-validation-service -v`
```

Any ideas?

Reply:
I would have to run it like this. Which is suboptimal.
```
$ RUST_LOG=info cargo run --bin mls-validation-service -- --version
   Compiling mls_validation_service v0.1.0 (/Users/martinkysel/work/libxmtp/mls_validation_service)
    Finished `dev` profile [unoptimized] target(s) in 1.29s
     Running `target/debug/mls-validation-service --version`
2024-10-21T13:29:30.722812Z  INFO mls_validation_service: Version: 0.1.0-a314006
```

Reply:
done!